docs/getting-started.md

```diff
@@ -32,9 +32,9 @@
 - [kubectl]
 - [kustomize]
 - make
-- gettext (with `envsubst` in your PATH)
+- [gettext](https://www.gnu.org/software/gettext/) (with `envsubst` in your PATH)
 - md5sum
-- bazel
+- [bazel](https://docs.bazel.build/versions/master/install-os-x.html#install-on-mac-os-x-homebrew)
```

Comment:
should we keep the link general for users who aren't on mac os (eg. Linux)?

Reply:
True, I can update!

Reply:
/hold